Pepperdash Core/Pepperdash Core/Comm/GenericSshClient.cs

```diff
@@ -473,28 +487,39 @@ void OnConnectionChange()
 		/// <param name="text"></param>
 		public void SendText(string text)
 		{
-			try
-			{
-                if (Client != null && TheStream != null && IsConnected)
-                {
-                    if (StreamDebugging.TxStreamDebuggingIsEnabled)
-                        Debug.Console(0, this, "Sending {0} characters of text: '{1}'", text.Length, ComTextHelper.GetDebugText(text));
-
-                    TheStream.Write(text);
-                    TheStream.Flush();
+		    try
+		    {
+		        if (Client != null && TheStream != null && IsConnected)
+		        {
+		            if (StreamDebugging.TxStreamDebuggingIsEnabled)
+		                Debug.Console(0,
+		                              this,
+		                              "Sending {0} characters of text: '{1}'",
+		                              text.Length,
+		                              ComTextHelper.GetDebugText(text));
+
+		            TheStream.Write(text);
+		            TheStream.Flush();
+		        }
+		        else
+		        {
+		            Debug.Console(1, this, "Client is null or disconnected.  Cannot Send Text");
+		        }
+		    }
+		    catch (ObjectDisposedException ex)
+            {
+                Debug.Console(0, this, "Exception: {0}", ex.Message);
+                Debug.Console(0, this, "Stack Trace: {0}", ex.StackTrace);
 
-                }
-                else
-                {
-                    Debug.Console(1, this, "Client is null or disconnected.  Cannot Send Text");
-                }
-			}
+		        KillClient(SocketStatus.SOCKET_STATUS_CONNECT_FAILED);
+                ReconnectTimer.Reset();
+		    }
 			catch (Exception ex)
 			{
-			    Debug.Console(0, "Exception: {0}", ex.Message);
-			    Debug.Console(0, "Stack Trace: {0}", ex.StackTrace);
+                Debug.Console(0, this, "Exception: {0}", ex.Message);

```

Comment:
Let's make both of these statements go to the error log, but be 0 for the message, and 1 for the stack trace